DESIGN.md

```
@@ -0,0 +1,448 @@
+### The flow
+
+```bash
+git clone https://github.com/TEDI-Design-System/react.git
+cd react
+nvm use            # Node >= 24, npm >= 11
+npm ci
+```
+
+Then, in Claude Code inside the repo:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)